src/components/CopyToClipboard/index.tsx

```diff
@@ -132,22 +128,9 @@ export const CopyAddressToClipboard: FC<CopyAddressToClipboardProps> = ({
   copyButtonPosition,
   chain,
   withFullAddress = false,
-  withLinkToBlockExplorer = false,
```

Comment:
Let's remove `withLinkToBlockExporer` from `CopyAddressClipboard` render in `MakeDeposit.tsx` and `UnmintDetils.tsx`. We should also remove it from `CopyAddressToClipboardProps` above. Right now it produces a warning in the console:

```
React does not recognize the `withLinkToBlockExplorer` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `withlinktoblockexplorer` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
Ref commit: 7e6f3cb7